src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1120,6 +1131,144 @@ protected static ImmutableArray<string> CheckTupleElementNames(int cardinality,
         public bool HasImplicitConversion(ITypeSymbol fromType, ITypeSymbol toType)
             => fromType != null && toType != null && this.ClassifyCommonConversion(fromType, toType).IsImplicit;
 
+        /// <summary>
+        /// Checks if <paramref name="symbol"/> is accessible from within <paramref name="within"/>. An optional qualifier of type
+        /// <paramref name="throughType"/> is used to resolve protected access for instance members. All symbols are
+        /// required to be from this compilation or some assembly referenced (<see cref="References"/>) by this
+        /// compilation. <paramref name="within"/> is required to be an <see cref="ITypeSymbol"/> or <see cref="IAssemblySymbol"/>.
```

Comment:
>ITypeSymbol [](start = 83, length = 11)

This comment does not match the diagnostic ("ITypeSymbol" vs. "INamedTypeSymbol")